src/util.js

```diff
@@ -265,9 +265,24 @@ export function chartify(results, description, requestParams) {
         return acc;
       }, {});
     }
+
     if (typeof obj === 'string' && obj.startsWith('@')) {
-      const column = obj.substring(1);
-      return results.map((entry) => entry[column]);
+      // @columnX,@columnY to be replaced with an array of {x, y} objects
+      return results
+        .map((entry) => obj
+          // take each segment
+          .split(',')
+          .map((e, i) => ({
+            key: String.fromCharCode(120 + i),
+            column: e.substring(1),
+          }))
+          .reduce((acc, { key, column }) => {
+            // eslint-disable-next-line no-param-reassign
+            acc[key] = entry[column];
+            return acc;
+          }, {}))
+        // if it is a single value, then just return that value
+        .map((entry) => (Object.keys(entry).length === 1 ? entry.x : entry));
```

Comment:
```suggestion
        .map((entry) => (Object.keys(entry).length === 1 && entry.hasOwnProperty('x') ? entry.x : entry));
```

Reply:
@dbrrt eslint hated this suggestion https://app.circleci.com/pipelines/github/adobe/helix-run-query/3482/workflows/f6ddaeb9-7920-4389-8b07-9acc6f4a349a/jobs/5870?invite=true#step-105-65_34